packages/common/src/videos/connector/index.ts

```diff
@@ -0,0 +1,11 @@
+import queryString from 'query-string';
+import AxiosInstance from "../../config/axios/configAxios";
+import {IFilterVideoRequest} from '../models';
+
+
+export const getCategories = async () => AxiosInstance.get("/videos/category")
+
+//TODO:ładny komentarz ?category=${categories}?title=${title}
```

Comment:
Dajcie znać kto stworzył taki url z parametrami  ?category=${categories}?title=${title}

Reply:
z tego co widzę to jest to podpisane jako Mikołaj :)

Reply:
bardzo prawdopodobne że ja a co tam ?